packages/abi-coder/src/types/JsonAbiNew.ts

```diff
@@ -91,7 +91,7 @@ export interface LoggedType {
 }
 
 export interface MessageType {
-  readonly message_id: string;
+  readonly messageId: string;
```

Comment:
The spec says the following: `"message_id": a unique string ID.` We should fix the spec given that this is the correct name.

Reply:
A rename such as this would be a `minor` breaking change, but in this case, we can safely push it as a `patch` since this prop didn't exist and was merged in a broken state, meaning nobody should be using it. Makes sense?

Reply:
Also, we don't need to wait for the spec to be updated to move on with this.